### ISSUE TITLE
Mind Controlling the executioner in Zul'Farrak causes him to not...

### ISSUE THREAD

**Issue by kaytotes** [Project-Epoch/bugtracker#8273]
> Mind Controlling the executioner in Zul'Farrak causes him to not drop the key basically bricking the instance

.go xyz 1838.080000 1149.060000 17.038700 209 4.055420

**Comment by Bennylavaa** [comment]
I cannot reproduce this, even after MC he drops the key

**Comment by Bennylavaa** [comment]
maybe there is more too it? I suspect from other reports he died with mind controlled which would result in no loot. TBH just make him so he cannot be mind controlled.

**Comment by Shadownflame** [comment]
> maybe there is more too it? I suspect from other reports he died with mind controlled which would result in no loot. TBH just make him so he cannot be mind controlled.

Yep, if a mob dies while mind controlled, they will drop no loot, that is how it works. However, if this mob is causing problems in particular I agree it would be the easiest solution to prevent MC.

**Comment by Shadownflame** [comment]
npc id 7274
